IZP2tester_zs2020.py

```diff
@@ -18,87 +19,87 @@
 cbLGRAY = '\033[1;37m'
 cbCYAN = '\033[1;46m'
 
+class TestResult:
+    SEGFAULT = "segfault"
+    ERROR = "error"
+    BINARY = "binary"
+    TIMEOUT = "timeout"
+    OK = "ok"
+    MEM_ERROR = "memerror"
+
+    def __init__(self, type, msg=''):
+        self.type = type
+        self.msg = msg
+
+def _run_test(executable, args, filename, memcheck=False, maxstack=False):
+    tmp_file_name = filename + '.tmp'
+    arguments = (['valgrind', '--log-fd=1', '-q', '--leak-check=full'] if memcheck else []) + \
```

Comment:
Can you wrap this in parentheses instead of using backslashes? (Let's try to be a bit pythonic here)